charts/rpe-service-auth-provider/values.template.yaml

```
@@ -0,0 +1,10 @@
+java:
+  applicationPort: 8489
+  environment:
+    TEST_SERVICE_SECRET : ${TEST_SERVICE_SECRET}
+    JWT_KEY : ${JWT-KEY}
```

needs the other side done too and in the Jenkinsfile_CNP
```suggestion
    JWT_KEY : ${JWT_KEY}
```